Src/DependencyCollector/Shared/Implementation/HttpDesktopDiagnosticSourceListener.cs

```diff
@@ -83,6 +83,15 @@ public void OnNext(KeyValuePair<string, object> value)
                         break;
                     }
 
+                    case "System.Net.Http.InitializationFailed":
+                    {
+                        DependencyTableStore.IsDesktopHttpDiagnosticSourceActivated = false;
+
+                        Exception ex = (Exception)value.Value.GetType().GetProperty("Exception")?.GetValue(value.Value);
+                        DependencyCollectorEventSource.Log.HttpHandlerDiagnosticListenerFailsToInitialize(ex?.ToInvariantString());
```

Comment:
What happens to the formatting call in the event source message if ex is null and null is passed?